README.md

```diff
@@ -9,24 +9,27 @@ _Having problems? want to contribute? join our [community slack](http://devtools
 
 > Generate a CHANGELOG from git metadata
 
-It's recommended you use the high level [standard-version](https://github.com/conventional-changelog/standard-version) library, which handles the entire tagging and CHANGELOG
-generation process.
-
-Alternatively, the [conventional-changelog-cli](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) allows you to generate a CHANGELOG from the
-command line.
-
 ## About this Repo
 
 The conventional-changelog repo is managed as a [monorepo](https://github.com/babel/babel/blob/master/doc/design/monorepo.md); it's composed of many npm packages.
 
 The original `conventional-changelog/conventional-changelog` API repo can be
 found in [packages/conventional-changelog](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog).
 
+## Getting started
```

Comment:
If someone lands on this GitHub repository, would they be more interested in how we organize our repository (Discussed in the _About this Repo_ section above), or how to use _conventional_ changelogs?

If the latter, can this section be moved up in the README?

Reply:
I agree, I think we should move it up